sr_object_tracking/launch/tracking_camera.launch

```diff
@@ -1,5 +1,7 @@
 <launch>
```

Comment:
Well, no.. I separated my launch file in two : one which uses the camera (this one) and the other which doesn't (tracking_no_camera). It's the last one that should be used with autopic then. 
Wasn't it the idea ?